### PR TITLE
Update Known issues

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #https://github.com/GoogleChromeLabs/Extension-Docs/issues/185

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Review by sebastianbenz — APPROVED** [review]

File: site/en/docs/extensions/migrating/known-issues/index.md [hunk 1]
```diff
@@ -24,25 +24,22 @@ Recently, we announced changes to the Manifest V2 deprecation timeline, and whil
...
-    **Work in progress:** `GEOLOCATION` added in Chrome 116 
 1. **Support File Handling API on ChromeOS** as a replacement for [`chrome.fileBrowserHandler`](/docs/extensions/reference/fileBrowserHandler/).
+1. **User Script support:** Allow registering content scripts with arbitrary code with the new [userScripts API](/docs/extensions/reference/userscripts/). (See [our proposal](https://github.com/w3c/webextensions/blob/main/proposals/user-scripts-api.md) for details.)
 
 These issues have been collected based on feedback from partners, bug reports, and developers. In addition to these, we will continue our ongoing work to address stability issues and improve overall performance. 
 
 The following issues have recently been addressed:
 
+1. Additional **strong service worker keepalives** for certain operations taking longer than five minutes.
+    * Added in Chrome 116 for `permissions.request()`, `desktopCapture.chooseDesktopMedia`, `identity.launchWebAuthFlow()` and `management.uninstall()`.
```
Comment (jpmedley) [inline site/en/docs/extensions/migrating/known-issues/index.md:35]:
```suggestion
    * Added in Chrome 116 for `permissions.request()`, `desktopCapture.chooseDesktopMedia()`, `identity.launchWebAuthFlow()` and `management.uninstall()`.
```

File: site/en/docs/extensions/migrating/known-issues/index.md [hunk 2]
```diff
@@ -24,25 +24,22 @@ Recently, we announced changes to the Manifest V2 deprecation timeline, and whil
...
+    * Added in Chrome 118 for `chrome.debugger`
+1. **Increase the number of static and enabled rulesets** for Declarative Net Request (DNR). Enabled static rulesets increased from 10 to 50 and total static rulesets from 50 to 100 [Chrome 120].
```
Comment (jpmedley) [inline site/en/docs/extensions/migrating/known-issues/index.md:37]:
```suggestion
1. **Increase the number of static and enabled rulesets** for Declarative Net Request. Enabled static rulesets increased from 10 to 50 and total static rulesets from 50 to 100 (Chrome 120).
```

File: site/en/docs/extensions/migrating/known-issues/index.md [hunk 3]
```diff
@@ -24,25 +24,22 @@ Recently, we announced changes to the Manifest V2 deprecation timeline, and whil
...
+1. Extend **[Offscreen document](/docs/extensions/reference/offscreen/) functionality** to support more reasons for using an offscreen document. Added [`GEOLOCATION`](/docs/extensions/mv3/geolocation/) in Chrome 116.
 1. Improving **support for the [`chrome.tabCapture`](/docs/extensions/reference/tabCapture/) API** [Chrome 116]:
     * Support calling `getMediaStreamId()` from a service worker.
     * Support obtaining a `MediaStream` from a stream ID in an offscreen document.
-1. **Extending service worker lifetimes** while there are active `WebSocket` connections [Chrome 116].
-
+1. **Extending service worker lifetimes** while there are active [`WebSocket` connections](/docs/extensions/mv3/tut_websockets/) [Chrome 116].
```
Comment (jpmedley) [inline site/en/docs/extensions/migrating/known-issues/index.md:42]:
```suggestion
1. **Extending service worker lifetimes** while there are active [`WebSocket` connections](/docs/extensions/mv3/tut_websockets/) (Chrome 116).
```

**Review by jpmedley — APPROVED** [review]
There are a few nits to commit. Ping me on chat for merge.

File: site/en/docs/extensions/migrating/known-issues/index.md [hunk 4]
```diff
@@ -67,3 +64,10 @@ The following issues have recently been addressed:
 **A:** We are confident that most request blocking use cases can be solved with the new [`declarativeNetRequest` API](/docs/extensions/reference/declarativeNetRequest/), which has the added benefit of avoiding the performance overhead of interprocess communication, executing code on every request, or requiring an active extension process at the time of the request. However, for complex [enterprise (or education) use cases](https://support.google.com/chrome/a/answer/9296680?hl=en), dynamic request blocking is still supported.
 
 **Did we miss something?** [Please let us know](https://groups.google.com/a/chromium.org/g/chromium-extensions).
+
+
+
+
+Preview
+
+
```
Comment (AmySteam) [inline site/en/docs/extensions/migrating/known-issues/index.md:73]:
```suggestion
```

**Pull request merged**